packages/ComponentLibrary/src/components/Table/TabNavigation/TabContent.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Typography } from '@mui/material';
+import { TabContentProps } from '../../../../../storybook/src/stories/Components/Table/types';
```

Comment:
hmmm this doesn't look good.
As far as I can understand, the `TabContentProps` type should be part of the ComponentLibrary